mlir-sycl/lib/Conversion/SYCLToLLVM/SYCLToLLVM.cpp

```diff
@@ -271,6 +271,15 @@ static Optional<Type> convertRangeType(sycl::RangeType type,
                          type.getBody(), converter);
 }
 
+/// Converts SYCL RoundedRangeKernel type to LLVM type.
```

Comment:
These `convertXXXType` declarations are not in alphabetical order. Can you move declarations around and put them in alphabetical order pls.

Reply:
Not sure if this will get in, so created https://github.com/intel/llvm/pull/7710.